src/routes/Route.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Routes,Route } from "react-router-dom";
+import { HomePage, VideoListing } from "../page";
+
+export const RoutePath = () => {
+  return (
+    <Routes>
+      <Route path="/" element={<HomePage />} />
+
+      <Route path="videolist" element={<VideoListing />} />
```

Comment:
i think it should be /videolist in the path.

Reply:
ok will do